packages/pluginsdk/package.json

```diff
@@ -33,15 +33,9 @@
     "yargs": "^16.2.0"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "17.1.0",
-    "@rollup/plugin-node-resolve": "11.1.1",
-    "@rollup/plugin-typescript": "8.1.1",
     "@spinnaker/core": "0.0.549",
+    "@spinnaker/scripts": "file:../scripts",
```

Comment:
this is going to cause problems for plugins, I assume this is temporary?

Reply:
oh... maybe it won't cause problems because it's a `devDependency`?

Reply:
Yep, this is temporary. I'll publish it right after merging this PR.

Reply:
> but... I think it needs to be an actual dependency so that it gets installed in a plugin when they install the pluginsdk.

Missed this comment. I'm going to test it out real quick.

Reply:
Yep, that's right. I'm fixing it now. I had tested it locally in a plugin, but I guess I would have moved it over to dev dependency later :(